SConstruct

```diff
@@ -198,13 +207,19 @@ add_sources(sources, "src/core", "cpp")
 add_sources(sources, "src/variant", "cpp")
 sources.extend([f for f in bindings if str(f).endswith(".cpp")])
 
-env["arch_suffix"] = env["arch"]
+suffix = ".{}.{}".format(env["platform"], env["target"])
+if env.dev_build:
+    suffix += ".dev"
+suffix += "." + env["arch"]
```

Comment:
Upstream Godot also has a `.double` suffix added between `.dev` and `.<arch>` for `float=64`, should we add it too?